assignments/static/assignments/detail.js

```diff
@@ -424,8 +424,13 @@ if (chart) {
     var ctx = chart.getContext('2d');
 
     const all_grades = JSON.parse(document.getElementById('all_grades').textContent);
+    // find the min and max of the grades from all_grades
+    const values = Object.values(all_grades);
+    // get the min and max of the grades
+    const minm = Math.min(...values);
+    const maxm = Math.max(...values);
```

Comment:
I am getting infinity on the x axis when there are no graded submissions. I think `minm` should be always set to 0 and `maxm` should be the max/possible points of the assignment